templates/e-commerce/blog-article.amp.html

```diff
@@ -0,0 +1,55 @@
+<!doctype html>
+
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<html ⚡>
+{{#header}}{{> ../../utils/amp-page-head.snip.html}}
+
+<body>
+  {{> ../../components/navbar/navbar.snip.html}} {{/header}}
+
+  <main id="content" role="main" class="main flex flex-wrap commerce-panel-filled">
+    {{#blog.articles.0}}
+    <article class="col-12 md-col-8 commerce-blog-wrapper pt2 md-pt4">
+      <amp-img class="md-col-10" src="{{images.0.narrow}}" srcset="{{images.0.narrow}} 750w,{{images.0.wide}} 1280w" width="{{imageWidth}}"
+        height="{{imageHeight}}" layout="responsive" alt="{{title}}"></amp-img>
+      <div class="md-col-10">
+        <div class="px2 mt3">
+          <h1 class="h3">{{title}}</h1>
+          <time datetime="{{datetime}}" class="block mb2">{{date}}</time>
+          <p class="mb5">{{paragraphs.0}}</p>
+        </div>
+      </div>
+      <amp-img class="mb3 md-col-10" src="{{images.1.narrow}}" srcset="{{images.1.narrow}} 750w,{{images.1.wide}} 1280w" width="{{imageWidth}}"
+        height="{{imageHeight}}" layout="responsive" alt="{{title}}"></amp-img>
+      <amp-img class="mb5 md-col-10" src="{{images.2.narrow}}" srcset="{{images.2.narrow}} 750w,{{images.2.wide}} 1280w" width="{{imageWidth}}"
+        height="{{imageHeight}}" layout="responsive" alt="{{title}}"></amp-img>
+      <div class="md-col-10">
+        <div class="px2 mb4">
+          <p>{{paragraphs.1}}</p>
+          <blockquote class="commerce-blockquote md-col-8">{{quote.text}}</blockquote>
+          <div class="commerce-blockquote-quoter">- {{quote.quoter}}</div>
```

Comment:
is this different from ampstart-blockquote? 
I mean not visually - functionally is this different?
If not we would need to rewrite the ampstart-blockquote on your page.